bin/ci/az-publish.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+source bin/ditto.sh
+
+set -eo pipefail
+CONFIG=Debug
+
+# $1 => SOURCE PATH
+# $2 => TARGET NAME
+function azupload {
+    echo "Dry run: ${1} published with name: ${2}"
+# az storage blob upload \
+#   --container-name ios-test-cloud-agent \
+#   --if-none-match "*" \
```

Comment:
I think it is acceptable to overwrite if there is an existing file.

Reply:
Done